main.go

```diff
@@ -204,6 +206,19 @@ func runCLIApp(c *cli.Context) (err error) {
 	config.OverrideWithLoggingFlags(mountConfig, flags.LogFile, flags.LogFormat,
 		flags.DebugFuse, flags.DebugGCS, flags.DebugMutex)
 
+	// if metadata-cache:ttl-secs has been set in config-file, then
+	// switch metadata cache ttls (stat-cache-ttl and type-cache-tll)
+	// to that as that takes precedence over both stat-cache-ttl and
+	// type-cache-tll.
+	if mountConfig.MetadataCacheConfig.TtlInSeconds != config.TtlInSecsUnset {
+		// if ttl-secs is set to -1, set StatOrTypeCacheTTL to the max possible duration.
+		if mountConfig.MetadataCacheConfig.TtlInSeconds == -1 {
+			flags.StatOrTypeCacheTTL = time.Duration(math.MaxInt64)
```

Comment:
multiply by time.Second ?

Reply:
No, StatOrTypeCacheTTL is time.Duration . So, no need to further multiply it with anything. `time.Duration(math.MaxInt64)` is the next duration represetable by time.Duration, ~290 years.